st2common/st2common/operators.py

```diff
@@ -169,8 +182,10 @@ def nexists(value, criteria_pattern):
     return value is None
 
 # operator match strings
-MATCH_WILDCARD = 'matchwildcard'
```

Comment:
generally not a good idea to remove operators. This will cause someones rule etc to break.